multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -24,6 +24,10 @@ externalDNSName: {{.ExternalDNSName}}
 # it for you.
 #hostedZone: ""
```

Comment:
Per my previous comment, if we are deprecating this flag, we should allow the user to *not* specify it. Then also add a comment here noting that this flag is deprecated in favor of `hostedZoneID`